ios/Exponent/Versioned/EXDefines.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+#import <Foundation/Foundation.h>
+
+#define LAST_SUPPORTED_SDK 33
+
+#define EX_REMOVE_ONCE_SDK_IS_PHASED_OUT(sdk) _Static_assert(sdk >= LAST_SUPPORTED_SDK, "SDK dropped, remove this code");
```

Comment:
I'd call it `EX_DEPRECATED_SINCE(sdk)` and maybe also put the SDK version to the assert message.
```suggestion
#define EX_DEPRECATED_SINCE(sdk) _Static_assert(sdk >= LAST_SUPPORTED_SDK, "SDK #sdk has been dropped, remove this code.");
```

Reply:
I don't like `EX_DEPRECATED_SINCE`, because it induces that thing marked by this is only deprecated, whereas we wanted something that would crash whole compilation once condition is satisfied 🤔